service/ui_plugin_build_baron.go

```diff
@@ -109,6 +110,23 @@ func (uis *UIServer) bbGetCreatedTickets(w http.ResponseWriter, r *http.Request)
 	gimlet.WriteJSON(w, results)
 }
 
+func (uis *UIServer) bbGetCustomCreatedTickets(w http.ResponseWriter, r *http.Request) {
+	taskId := gimlet.GetVars(r)["task_id"]
+	var results []annotations.IssueLink
+
+	annotations, err := annotations.FindByTaskId(taskId)
```

Comment:
Do we not want to filter this by task execution?

Reply:
I didn't because build baron created tickets don't.

Reply:
I'm still a bit suspicious but I suppose we can see if it becomes a problem; I'm not sure why we would or wouldn't use execution really.